### PR TITLE
Add Python script for folding stacktrack profling samples for flamegraph

### PR DESCRIPTION
**Pull request by viirya**
Thanks for open sourcing `jvm-profiler` and it is useful for profiling Spark applications.

We are trying to generate flamegraph from Stacktrack Profiling output. This is a simple Python script for folding stack samples. We think it may be useful for others.

### CODE REVIEW

File: stackcollapse.py [hunk 1]
```diff
@@ -0,0 +1,32 @@
...
+
+    for line in f.readlines():
+        stacktraceLog = json.loads(line)
+
+        assert 'stacktrace' in stacktraceLog, "Malformated json. 'stacktrace' key doesn't exist."
+        stacktrace = stacktraceLog['stacktrace']
+        assert 'count' in stacktraceLog, "Malformated json. 'count' key doesn't exist."
+        count = stacktraceLog['count']
+
+        if len(stacktrace) == 0:
```
Comment (felixcheung) [inline stackcollapse.py]: maybe move this check and continue before L19?

Comment (viirya) [reply stackcollapse.py]: Sure. Thanks.

**Comment by felixcheung** [comment]
LGTM,
hold for another day for review, thx

**Comment by viirya** [comment]
@felixcheung @boy-uber Thanks for review and merging!

**Pull request merged**